api/src/main/java/io/opencensus/trace/unsafe/ContextHandleUtils.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016-17, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.trace.unsafe;
+
+import io.opencensus.internal.Provider;
+import io.opencensus.trace.ContextHandle;
+import io.opencensus.trace.ContextManager;
+import io.opencensus.trace.Span;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.annotation.Nullable;
+
+public class ContextHandleUtils {
+
+  // No instance of this class.
+  private ContextHandleUtils() {}
+
+  private static final Logger LOGGER = Logger.getLogger(ContextHandleUtils.class.getName());
+  private static final ContextManager CONTEXT_MANAGER =
+      loadContextManager(ContextManager.class.getClassLoader());
+
+  private static ContextManager loadContextManager(@Nullable ClassLoader classLoader) {
+    try {
+      return Provider.createInstance(
+          Class.forName(
+              "io.opentelemetry.opencensusshim.OpenTelemetryContextManager",
+              /*initialize=*/ true,
+              classLoader),
+          ContextManager.class);
+    } catch (ClassNotFoundException e) {
+      LOGGER.log(
+          Level.FINE,
+          "Couldn't load full implementation for OpenTelemetry context manager, now loading "
+              + "original implementation.",
+          e);
+    }
+    return new ContextManagerImpl();
+  }
```

Comment:
Loading the migration library context manager by reflection here so applications migrating only have to add the new library dependency instead of adding new code.
More info here: https://github.com/zoercai/opentelemetry-operations-java/pull/3/files/7029c849e2f0e90c5178e044043e4e9aa0c68bda#r505253778

Reply:
LGTM.